### PR TITLE
Add omit_accounts_and_delegations option to GetStakingByHeight

### PR DESCRIPTION
**Pull request by christophermanning**
With omit_accounts_and_delegations=true, the request takes 0.094656
seconds, without it, it takes 16.777716 seconds

### CODE REVIEW

File: test/integration/staking_test.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	stateServer := statepb.NewStateServiceClient(conn)
+
+	start = time.Now()
+	data, err := stateServer.GetStakingByHeight(ctx, &statepb.GetStakingByHeightRequest{Height: int64(height), OmitAccountsAndDelegations: true})
+	fmt.Printf("GetStakingByHeight with omit duration: %f\n", time.Since(start).Seconds())
```
Comment (lukanus) [inline test/integration/staking_test.go:40]: FYI There is a nice lib for output formatting [zaptest ](https://pkg.go.dev/go.uber.org/zap/zaptest) :)

File: test/integration/staking_test.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+	if err != nil {
+		panic(err)
```
Comment (lukanus) [inline test/integration/staking_test.go:42]: t.Error() :)

File: go.mod [hunk 3]
```diff
@@ -3,14 +3,13 @@ module github.com/figment-networks/oasis-rpc-proxy
 go 1.17
 
 replace (
+	github.com/google/flatbuffers => github.com/google/flatbuffers v1.12.1
```
Comment (lukanus) [inline go.mod:6]: probably not used

File: grpc/state/statepb/state.proto [hunk 4]
```diff
@@ -188,6 +188,9 @@ message GetByHeightResponse {
 
 message GetStakingByHeightRequest {
     int64 height = 1;
+
+    // omitting the accounts and delegations decreases response time
+    bool omit_accounts_and_delegations = 2;
```
Comment (lukanus) [inline grpc/state/statepb/state.proto:193]: I know it should probably work fine but have you checked if calling that with 
```message GetStakingByHeightRequest {
    int64 height = 1;
    }
```
still works fine?

**Review by lukanus — APPROVED** [review]
Looks good! 
The only important one is if the additive change for protobuf message won't break the older version of api.

File: go.mod (hunk 3 above)
Comment (christophermanning) [reply go.mod:6]: this line is just moved from running `go mod tidy` (the diff shows this same line deleted from a few lines below)

File: grpc/state/statepb/state.proto (hunk 4 above)
Comment (christophermanning) [reply grpc/state/statepb/state.proto:193]: yes, it works with only specifying height, the `bool` defaults to false.

I ran that query in the test file: https://github.com/figment-networks/oasis-rpc-proxy/pull/17/files#diff-b8f9f9a3c283bb8972b19cf271bfa5a03155ef686b722e5ac8a925c973f31e84R59

**Pull request merged**